src/system/NewTabs/Tabs.js

```diff
@@ -0,0 +1,28 @@
+/** @jsxImportSource theme-ui */
+
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import * as TabsPrimitive from '@radix-ui/react-tabs';
+
+/**
+ * Internal dependencies
+ */
+
+const NewTabs = ( { defaultValue = null, className = null, sx, ...props } ) => (
```

Comment:
It would be nice to add the `forwardRef` capability, and pass it to the first container.

Example:

```
const Card = React.forwardRef( ( { variant = 'primary', sx = {}, className, ...props }, ref ) => {
	return (
		<Box
			ref={ ref }
```